modules/nuxeo-apidoc-repo/src/test/java/org/nuxeo/apidoc/test/AbstractApidocTest.java

```diff
@@ -142,8 +142,8 @@ protected void checkJsonAssertEquals(String path, String actualContent) throws E
      *
      * @since 20.0.0
      */
-    protected static Function<String, String> getJsonTestUpdater() {
-        Function<String, String> function = (String s) -> {
+    protected static UnaryOperator<String> getJsonTestUpdater() {
+        return (String s) -> {
```

Comment:
This method returning a lambda could probably be converted to just the method with the code and used as a method reference.

Reply:
I don't understand

Reply:
You can define 

    protected static String testTransformer(String s) {
        String res = "";
        // deal with manifests (...)
        String[] lines = s.split("\n");    }
        ...
        return res;
    }

And call it with 

    checkContentEquals(path, actualContent, UPDATE_REFERENCE_FILES_ON_FAILURE, false, this::testTransformer);

and

    actualContent = testTransformer(actualContent);

Reply:
I don't remember why I made this a function at some point (maybe to be able to override it from subclasses when not static, don't know). I changed it here because of some warning on sonarlint but I'd rather avoid touching it at all if refinements are needed: please tell if i should revert the changes instead.

Reply:
As you wish this is a detail